cookbooks/aws-parallelcluster-environment/spec/unit/libraries/volume_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe 'aws-parallelcluster-environment:libraries:format_directory' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new.converge_dsl do
+      volume 'nothing' do
```

Comment:
Not sure this is even required, I guess you can remove the body of the converge.

Reply:
Thanks, I'll address it in a separate patch.